src/main/scala/com/fulcrumgenomics/sv/BreakpointEvidence.scala

```diff
@@ -26,13 +36,12 @@ object BreakpointEvidence {
  *
  * @param breakpoint the breakpoint for which we have evidence
  * @param evidence the type of evidence for this breakpoint
+ * @param from the [[SamRecord]](s) that provided evidence of this break point going "from" the breakpoint
+ * @param into the [[SamRecord]](s) that provided evidence of this break point going "into" the breakpoint
+ * @param fromIsLeft if the records in `from` correspond to the left side of the breakpoint, otherwise the right
  */
-case class BreakpointEvidence(breakpoint: Breakpoint, evidence: EvidenceType) extends Ordered[BreakpointEvidence] {
-
-  /** Defines an ordering over breakpoints, first by genomic coordinates of the left then right end, then evidence type. */
-  def compare(that: BreakpointEvidence): Int = {
-    var result              = this.breakpoint.compare(that.breakpoint)
-    if (result == 0) result = this.evidence.compare(that.evidence)
-    result
-  }
-}
+case class BreakpointEvidence(breakpoint: Breakpoint,
+                              evidence: EvidenceType,
+                              from: Set[SamRecord] = Set.empty,
+                              into: Set[SamRecord] = Set.empty,
+                              fromIsLeft: Boolean = true)
```

Comment:
I want a better name for this, but have no suggestion for a better name :/

Reply:
`lateral`?  To mean that left and from are on the same side.  Leaving for now since I don't either.